uiCore/src/main/res/values/themes.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="Theme.ForzenBook.Light" parent="Theme.AppCompat.DayNight.NoActionBar">
+        <item name="primary">@color/white</item>
```

Comment:
Attribute and dimen names are final as camel case also

Reply:
renamed them